Karma/Views/Home/DonateDialog.cshtml

```diff
@@ -48,15 +49,15 @@
             </div>
             <div class="row align-items-center">
                 <div class="col-12 text text-center">
-                    Your item is currently not on demand?
+                    @localizer["createPostPhrase"]
                 </div>
             </div>
             @*TODO: redirect to page (dialog?) for donate post creation*@
             <div class="row">
                 <button type="button" class="btn btn-light col">
                     <div class="row align-items-center">
                         <div class="col-12 text-center font-weight-bold">
-                            <a class="nav-link text-dark" asp-area="" asp-controller="Posts" asp-action="Donate">Create a donation post</a>
+                            @localizer["createPost"]
```

Comment:
The link would stop working and when button is pressed it won't redirect to `/Posts/Donate`.